### PR TITLE
Add exception to exclusive Qualified songs BN rule

### PR DESCRIPTION
**Pull request by pishifat**
discussion: https://osu.ppy.sh/community/forums/topics/979133

### CODE REVIEW

File: wiki/People/The_Team/Beatmap_Nominators/Rules/en.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Beatmap Nominators possess the tools to nominate beatmaps and shape the world of
 
 ### Qualification
 
-- **Do not qualify beatmapsets whose song already has a Qualified beatmapset, unless no game modes of both sets overlap—this includes remixed versions**. For example an osu!taiko and osu! beatmapset of the same song can exist in the Qualified section, whilst an osu!taiko and osu!/osu!taiko hybrid set cannot. This ensures variety among Ranked beatmaps within their respective game modes.
+- **Only one beatmap of a song may be qualified per game mode at a time. This includes different length cuts and simple covers of a song, but excludes remixes, up to three different versions of which may be qualified per game mode at a time.** This ensures variety among Ranked beatmaps within their respective game modes.
```
Comment (cl8n) [inline wiki/People/The_Team/Beatmap_Nominators/Rules/en.md]: - should use similar language as the other points, speaking directly to the bn. so first sentence could be "Do not qualify beatmapsets with songs already used in Qualified beatmaps of the same mode." ?
- "includes" and "excludes" could be interpreted to mean the opposite of your intention, I think that's why the original was confusing. "Cuts and covers do not count as different songs when applying this rule, but remixes do" or somethign similar that sounds better for second sentence 
- possibly remove the remixes thing altogether per The Dm
- can drop the "within their respective game modes" from last sentence cuz its obvious and just puts morewords in already wordy rule

Comment (pishifat) [reply wiki/People/The_Team/Beatmap_Nominators/Rules/en.md]: reworded

File: wiki/People/The_Team/Beatmap_Nominators/Rules/en.md [hunk 2]
```diff
@@ -22,7 +22,9 @@ Beatmap Nominators possess the tools to nominate beatmaps and shape the world of
 
 ### Qualification
 
-- **Do not qualify beatmapsets whose song already has a Qualified beatmapset, unless no game modes of both sets overlap—this includes remixed versions**. For example an osu!taiko and osu! beatmapset of the same song can exist in the Qualified section, whilst an osu!taiko and osu!/osu!taiko hybrid set cannot. This ensures variety among Ranked beatmaps within their respective game modes.
+- **To maintain a variety of songs in the Ranked section, there are two requirements:**
+    - Different beatmaps of similar versions of a song cannot be qualified in the same mode at the same time. Similar versions of a song include cuts, extensions and simple covers.
+    - Following the above, only 3 significantly different versions of a song may be qualified in the same mode at the same time. Significantly different versions would be remixes, compilations and mashups.
```
Comment (dorothy3242) [inline wiki/People/The_Team/Beatmap_Nominators/Rules/en.md]:
```suggestion
    - Only 3 significantly different versions of a song may be qualified in the same mode at the same time. Significantly different versions would be remixes, compilations and mashups.
```

**Comment by cl8n** [comment]
the bold is on the wrong part, it doesn't say anything by itself. bold first part of each bullet instead

**Comment by cl8n** [comment]
and probably just remove the first line too

**Comment by cl8n** [comment]
merge master into this branch if you're going to use it again

**Pull request merged**